test/Utilities/model.jl

```diff
@@ -182,7 +182,7 @@ function test_ObjectiveFunction()
     return
 end
 
-function _test_T
+function _test_T()
```

Comment:
Haha. I didn't consider this edge case.

Reply:
Was it caused by one of the regexes I assume?